src/java/com/vaklinov/zcashui/SendCashPanel.java

```diff
@@ -418,22 +418,20 @@ private void sendCash()
 			errorMessage = "Destination address is invalid; it is too long.";
 		}
 		
-		// Prevent accidental sending to non-ZEN addresses (which zend supports) probably because of
-		// ZClassic compatibility
+		// Prevent accidental sending to non-ZCL addresses (as seems to be supported by daemon)
 		if (!installationObserver.isOnTestNet())
 		{
       //TODO - ZCL
 			if (!(destinationAddress.startsWith("zc") || 
-				  destinationAddress.startsWith("zn") ||
-				  destinationAddress.startsWith("zs")))
+				  destinationAddress.startsWith("t1")))
```

Comment:
What about t3? https://github.com/z-classic/zclassic/blob/master/src/chainparams.cpp#L105

Reply:
(can do t3 in a separate PR, no worries)